runtime/vm/jnimem.c

```diff
@@ -51,6 +51,10 @@ void* jniArrayAllocateMemoryFromThread(J9VMThread* vmThread, UDATA sizeInBytes)
 #endif
 }
 
+void* jniArrayAllocateMemory32FromThread(J9VMThread* vmThread, UDATA sizeInBytes) {
```

Comment:
Should these functions (and their prototypes) be ifdeffed for consistency?

Reply:
@gacholio: The code consuming these two Memory32 APIs are in the refactored get/releaseArrayElements/StringImpl, that takes in the newly added `ensureMem32` parameter.  Arguably, these two are generic enough utility functions -- and if I #ifdeffed them out, the base implementation won't actually guarantee correctness if a caller passes true to `ensureMem32`.  Thoughts?

```C
+static void*
+getArrayElementsImpl(JNIEnv *env, jarray array, jboolean *isCopy, jboolean ensureMem32)
 {
....
              if (ensureMem32) {
                       elems = jniArrayAllocateMemory32FromThread(currentThread, ROUND_UP_TO_POWEROF2(byteCount, sizeof(UDATA)));
               } else {
                       elems = jniArrayAllocateMemoryFromThread(currentThread, ROUND_UP_TO_POWEROF2(byteCount, sizeof(UDATA)));
               }
```

Reply:
Fair enough, I suppose - leave them alone.